cli/src/config/revsets.toml

```diff
@@ -7,6 +7,9 @@ latest(
   remote_branches(exact:"main", exact:"origin") | 
   remote_branches(exact:"master", exact:"origin") | 
   remote_branches(exact:"trunk", exact:"origin") |
+  remote_branches(exact:"main", exact:"upstream") |
+  remote_branches(exact:"master", exact:"upstream") |
+  remote_branches(exact:"trunk", exact:"upstream") |
```

Comment:
Ah, I suppose people might clone from their own forks on GitHub (or similar) and then that would be called "origin", and then they add the upstream repo afterwards, which by convention is often called "upstream"? Makes sense